File: expectanim/src/main/java/com/github/florent37/expectanim/core/rotation/RotationExpectationOriginal.java
package com.github.florent37.expectanim.core.rotation;

import android.view.View;

/**
 * Created by florentchampigny on 17/02/2017.
 */
public class RotationExpectationOriginal extends RotationExpectation {

    @Override
    public Float getCalculatedRotation(View viewToMove) {
        return 0f;
    }

    @Override
    public Float getCalculatedRotationX(View viewToMove) {
        return 0f;
    }

    @Override
    public Float getCalculatedRotationY(View viewToMove) {
        return 0f;
    }
}

File: kotlinpleaseanimate/src/main/java/com/github/florent37/kotlin/pleaseanimate/core/rotation/RotationExpectationOriginal.kt
package com.github.florent37.kotlin.pleaseanimate.core.rotation

import android.view.View

class RotationExpectationOriginal : RotationExpectation() {

    override fun getCalculatedRotation(viewToMove: View): Float? {
        return 0f
    }

    override fun getCalculatedRotationX(viewToMove: View): Float? {
        return 0f
    }

    override fun getCalculatedRotationY(viewToMove: View): Float? {
        return 0f
    }
}
